index.html

```diff
@@ -4,11 +4,200 @@
+                      <button type="button" class="cta_model">Post</button>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <!-- model finish -->
+
+              <form
+                class="d-flex align-items-center justify-content-center"
+                role="search"
+              >
+                <input
+                  class="form-control me-2"
+                  type="search"
+                  placeholder="Search"
+                  aria-label="Search"
+                />
+                <button class="cta_search" type="submit">Search</button>
+              </form>
+            </div>
+          </div>
+        </nav>
+      </div>
+    </header>
+    <!-- header finished here main section start from her  -->
+    <main>
+      <div class="hero_section">
+        <div class="container">
+          <div class="row">
+            <div class="col-12 col-lg-6 col-md-6 col-sm-12">
+              <img
+                src="/image/pictures/pic-8.jpg"
+                alt="two person are looking talking and smiling and they have their electric cycles"
+                class="hero_img"
+              />
+              <h1 class="hero_heading">Find EL Bike partners</h1>
+            </div>
+            <div class="col-12 col-lg-6 col-md-6 col-sm-12">
+              <div class="name hero_section">
+                <img
+                  src="/image/profile/profile-pic 1.png"
+                  alt="profile image of lady"
+                  class="profile"
+                />
+                <span>Selly the biker</span>
+              </div>
+              <div class="back">
+                <p>Hi! i am free on sunday, should to go out on cycle tour.</p>
+              </div>
+              <div class="side">
+                <div class="back">
+                  <p>Hi! yes Selly sounds like a good plan.</p>
+                </div>
+                <div class="name">
+                  <img
+                    src="/image/profile/profile-pic 5.png"
+                    alt="profile image of man"
+                    class="profile"
+                  />
+                  <span>Bob the biker</span>
+                </div>
+              </div>
+              <div class="name hero_section">
+                <img
+                  src="/image/profile/profile-pic 6.png"
+                  alt="profile image of lady"
+                  class="profile"
+                />
+                <span>Rita the biker</span>
+              </div>
+              <div class="back">
+                <p>
+                  Hi! mee too free on sunday, can you make a group maybe we can
+                  ask more bikers to join?.
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+    <!-- main finished here footer section start from her  -->
+    <footer>
+      <div class="container">
+        <div class="row logo">
```

Comment:
The child of a row should always be a column class

Reply:
yes i changed it and added div and class col